client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/index.tsx

```diff
@@ -462,6 +477,8 @@ export default function NotificationSettings( {
 					recordEvent={ recordEvent }
 					selectedDuration={ selectedDuration }
 					selectDuration={ selectDuration }
+					showPaidDuration={ isDowntimeMonitoringPaidTierEnabled }
```

Comment:
Thanks for raising this, @mattgawarecki. I've initially been having second thoughts about adding the feature flag check within the `NotificationDuration`. In the upcoming PR, more components within the Notification Settings will rely on this flag, and I wanted to have a single place to check this. However, this will introduce more props later on, which now I realize would be worst code readability-wise.


I decided to take your suggestion as I realized that doing so makes the component props less confusing now that we have `enablePaidDurations` props. 👍